tests/Session/test_profile.py

```diff
@@ -43,7 +43,9 @@ def testGetOpenFoodStorage(self):
        
         # Should return not none if food storage is open. 
         self.profile.addFoodStorage(FREEZER, freezer_name)
+        freezer = self.profile.getFoodStorage(FREEZER, freezer_name)
```

Comment:
instead of calling `getFoodStorage()` you should be calling `getOpenFoodStorage()` because that's the method you're testing. Then you can use it in your asserts below.

Reply:
Done!